### PR TITLE
boards/arm64/qemu_cortex_a53: Synchronize defconfigs

### PR DESCRIPTION
**Pull request by hx-bora**
The various qemu_cortex_a53 defconfigs diverged.

We harmonize these files, and put the config-specific features at the end.

### CODE REVIEW

File: boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig [hunk 1]
```diff
@@ -1,7 +1,13 @@
 CONFIG_SOC_QEMU_CORTEX_A53=y
 CONFIG_BOARD_QEMU_CORTEX_A53=y
 CONFIG_ARM_ARCH_TIMER=y
-CONFIG_QEMU_ICOUNT=n
+CONFIG_ARMV8_A_NS=y
+CONFIG_QEMU_ICOUNT=y
+CONFIG_QEMU_ICOUNT_SHIFT=4
+CONFIG_QEMU_ICOUNT_SLEEP=y
```
Comment (carlocaione) [inline boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig:7]: If we really really really do not need this, I'd leave this out.

File: boards/arm64/qemu_cortex_a53/qemu_cortex_a53_xip_defconfig [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
-# Enable XIP
-CONFIG_XIP=y
-
 # Enable console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
+# PSCI is supported
+CONFIG_PM_CPU_OPS=y
+
```
Comment (carlocaione) [inline boards/arm64/qemu_cortex_a53/qemu_cortex_a53_xip_defconfig:21]: ditto

**Review by carlocaione — CHANGES_REQUESTED** [review]

File: boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig (hunk 1 above)
Comment (carlocaione) [reply boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig]: for the XIP is probably simply forgotten. The XIP should closely follow what's done for `qemu_cortex_a53`.

File: boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig [hunk 3]
```diff
@@ -10,15 +12,22 @@ CONFIG_SERIAL=y
...
-
 # Enable serial port
 CONFIG_UART_INTERRUPT_DRIVEN=y
 
+# icount does not work well with SMP
+CONFIG_QEMU_ICOUNT=n
+
+# We have multiple QEMU-A53 boards,
+# so let us exercise NS on this one
+# (plus it is needed for SMP)
```
Comment (carlocaione) [inline boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig:23]: can you align this comment properly to 80 columns?

**Review by carlocaione — APPROVED** [review]

**Review by SgrrZhf — APPROVED** [review]
LGTM

**Review by povergoing — APPROVED** [review]

**Pull request merged**